src/protocol/jdbc/org/apache/jmeter/protocol/jdbc/AbstractJDBCTestElement.java

```diff
@@ -594,6 +576,39 @@ private String getStringFromResultSet(ResultSet rs) throws SQLException, Unsuppo
         return sb.toString();
     }
 
+    private int processRow(ResultSet rs, ResultSetMetaData meta, StringBuilder sb, int numColumns,
+            JMeterVariables jmvars, String[] varNames, List<Map<String, Object>> results, int j)
```

Comment:
Better name j, currentIterationIndex for example ?

Reply:
done